CHANGELOG.md

```diff
@@ -1,18 +0,0 @@
-## 1.1.0 [2023-02-28]
```

Comment:
Why delete this file?

Reply:
It seems we do not maintain it (it's 1.1.3 now). Further release notes can be covered in the GitHub Releases content.